akka-docs/src/main/paradox/cluster-client.md

```diff
@@ -230,12 +230,9 @@ contacts can be fetched and a new cluster client started.
 
 ## Migration to Akka gRPC
 
-Cluster Client is deprecated and it is not advised to build new applications with Cluster Client.
-We recommend using [Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/index.html) over using Cluster Client.
-
-Cluster Client is convenient to use if you have Akka actors on both client and server side, but
-a more decoupled, and therefore better, solution would be to use Akka gRPC and define
-application specific protocol buffer messages and gRPC service calls. The benefits are:
+Cluster Client is deprecated and it is not advised to build new applications with it.
+As a replacement we recommend using [Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/index.html)
+with an application-specific protocol. The benefits of this approach are:
```

Comment:
Alternatively we could say you could use Akka HTTP if gRPC isn't an accepted option in your environment, probably not worth it